Documentation/Storage-Configuration/Advanced/ceph-configuration.md

```diff
@@ -47,12 +47,24 @@ kubectl apply -f common.yaml -f operator.yaml -f cluster.yaml # add other files
 
 ## Deploying a second cluster
 
-If you wish to create a new CephCluster in a different namespace than `rook-ceph` while using a single operator to manage both clusters execute the following:
+If you wish to create a new CephCluster in a different namespace than `rook-ceph` while using a
+single operator to manage both clusters change `SECOND_ROOK_CLUSTER_NAMESPACE`. The `common-second-cluster`
+manifests is commented to allow for easy `sed` replacements.
 
 ```console
 cd deploy/examples
+export SECOND_ROOK_CLUSTER_NAMESPACE="rook-ceph"
 
-NAMESPACE=rook-ceph-secondary envsubst < common-second-cluster.yaml | kubectl create -f -
+sed -i.bak \
+    -e "s/\(.*\):.*# namespace:operator/\1: $ROOK_OPERATOR_NAMESPACE # namespace:operator/g" \
```

Comment:
Since this is a separate section, let's define again the ROOK_OPERATOR_NAMESPACE env var. They might not have defined it if they didn't follow the previous section.